examples/n-body/rust/README.md

```diff
@@ -1,7 +1,7 @@
 ### Build
 
 ```bash
-cargo build --release --target=wasm32-unknown-unknown
+cargo +nightly build --release --target wasm32-unknown-unknown
```

Comment:
That's nice. I'm using nightly by default and totally forgot that others use stable or beta by default.